llvm-spirv/lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1919,12 +1919,12 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
     MemSetInst *MSI = cast<MemSetInst>(II);
     Value *Val = MSI->getValue();
     if (!isa<Constant>(Val)) {
-      assert(!"Can't translate llvm.memset with non-const `value` argument");
```

Comment:
I suggest to move llvm-spirv changes in https://github.com/KhronosGroup/SPIRV-LLVM-Translator

Reply:
Yes, please. This repo contains a _copy_ of that project and generally we try to avoid any extra changes on top of the upstream repo. Once your patch is landed there it will be automatically synced back to intel/llvm (done on weekly basis)